packages/service-provider-core/src/uri-generator.spec.ts

```diff
@@ -255,6 +255,24 @@ describe('uri-generator.generate-uri', () => {
           expect(generateUri(options)).to.equal('');
         });
       });
+
+      context('when providing explicit serverSelectionTimeoutMS', () => {
+        const uri = 'mongodb://127.0.0.2/foo?serverSelectionTimeoutMS=10';
+        const options = { _: [uri] };
+
+        it('returns an empty string', () => {
```

Comment:
Nit: `returns an empty string` is probably copy-paste? 🤔

Reply:
Yes :)